src/pages/index.tsx

```diff
@@ -12,8 +49,97 @@ export default function Home() {
+              <button
+                type="button"
+                className="rounded-lg  bg-yellow-300 px-6 py-3.5 text-center text-base font-medium text-black hover:bg-blue-800 focus:outline-none focus:ring-4 focus:ring-blue-300 dark:bg-blue-600 dark:hover:bg-blue-700 dark:focus:ring-blue-800"
+              >
+                Join Now
+              </button>
+              <button
+                type="button"
+                className="rounded-lg bg-yellow-300 px-6 py-3.5 text-center text-base font-medium text-black hover:bg-blue-800 focus:outline-none focus:ring-4 focus:ring-blue-300 dark:bg-blue-600 dark:hover:bg-blue-700 dark:focus:ring-blue-800"
+              >
+                Extra large
+              </button>
+            </div>
+          </div>
+          <div className="mt-10 sm:mt-0">
+            <Carousel />
+          </div>
+        </div>
+
+        <div>
+          <section id="Programs" className="programs">
+            <div className="container-3">
+              <h1 className="centered-heading-3 margin-bottom-32px flex justify-center pt-60 text-4xl">
+                🧠 Programs
+              </h1>
+              <div className="hero-wrapper-2 flex flex-col justify-center sm:flex-row sm:items-center sm:pt-10 md:flex-row">
+                <p className="programs margin-bottom-16px text-left md:p-0 lg:pl-80 lg:pr-10">
+                  INIT chapters carry out our signature experiential learning
+                  and career development programs within their communities.
+                  These programs attract thousands of individuals annually,
+                  helping them gain technical skills, work on projects, improve
+                  their resume, prepare for interviews, connect with industry,
+                  and much more. &apos;
+                  <a
+                    href="https://2022.shellhacks.net/"
+                    target="_blank"
+                    rel="noopener noreferrer"
+                    className="text-span-3 text-yellow-200"
+                  >
+                    ShellHacks
+                  </a>
+                  , hosted by our INIT FIU chapter, is the largest hackathon in
+                  Florida and brings together 1,500 students to innovate each
+                  year.
+                </p>
+                <Image
+                  src="/assets/images/programs.png"
+                  loading="lazy"
+                  alt=""
+                  width={432}
+                  height={300}
+                  sizes="(max-width: 479px) 94vw, (max-width: 767px) 95vw, (max-width: 991px) 92vw, (max-width: 1439px) 43vw, 432.3984375px"
+                  className="gallery-image shadow-two flex lg:pr-60"
+                  style={{
+                    width: "40%", // Default width
+                    marginTop: "20px", // Add top margin to create space
+                  }}
+                />
+              </div>
```

Comment:
![image](https://github.com/dabarcenas6921/init-dashboard/assets/90057009/2c0f19be-bf99-48bb-b1a4-636911051209)

This is too wide right now. I would suggest making it the same width as the carousel and mini paragraph above for consistency.